### DISCUSSION TITLE
Alpha 178

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#6508] [Release Notes]
- Experimental sixel graphics support
- Added 'focus follows mouse' mode for pane navigation (Atte Timonen) [#6465](https://github.com/Eugeny/tabby/pull/6465)
- Added hotkeys for navigating to a specific pane (Atte Timonen) [#6465](https://github.com/Eugeny/tabby/pull/6465)
- Update icons for built-in Windows profiles (Liangcheng Juves) [#6414](https://github.com/Eugeny/tabby/pull/6414)
- Fix the problem that the `Developer Prompt for VS 2022` profile could not be loaded (Liangcheng Juves) [#6414](https://github.com/Eugeny/tabby/pull/6414)
- Added Ukrainian translation
- Improved flow control for better speed
- Fixed overview ruler alignment
- More prominent active search result highlighting
- macOS: accept first mouse click 
- Allow customizing selection foreground color
- Hide description in selector model if it's the same as the title (Brandon Rothweiler) [#6493](https://github.com/Eugeny/tabby/pull/6493)
- Allow directly opening script (.sh / .bat / .ps1) and .command files 

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.178'>Alpha 178</a>.</em>

**Comment by Otpvondoiats** [comment]
i can't start program on windows10. click
